<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>514</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_side2">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сторона 2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_side2"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_side1">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сторона 1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_angle">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Угол°</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_side1"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_angle"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_calculate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>20</width>
      <height>461</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Стереть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_S">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Площадь =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_P">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Периметр=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_P_answ">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_S_answ">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_chek">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
